Sources/WalletConnectSigner/Signer/MessageSigner.swift

```diff
@@ -9,16 +9,20 @@ public protocol MessageSignatureVerifying {
 }
 
 public protocol MessageSigning {
-    func sign(payload: AuthPayload,
-        address: String,
+    func sign(message: String,
+        privateKey: Data,
+        type: CacaoSignatureType
+    ) throws -> CacaoSignature
+
+    func sign(payload: CacaoPayload,
         privateKey: Data,
         type: CacaoSignatureType
     ) throws -> CacaoSignature
 }
 
-public typealias AuthMessageSigner = MessageSignatureVerifying & MessageSigning
+public typealias CacaoMessageSigner = MessageSignatureVerifying & MessageSigning
```

Comment:
same here CacaoMessageSigner couples this to cacao object but it doesn't have to be wrapped into cacao

Reply:
same